uuv_dave/launch/valve_manipulation_demo.launch

```diff
@@ -0,0 +1,83 @@
+<!--
+UUV launch for project DAVE
+-->
+<launch>
+    <arg name="gui" default="true"/>
+    <arg name="paused" default="false"/>
+    <arg name="world_name" default="$(find uuv_dave)/worlds/uuv_dave_ocean_waves.world"/>
+    <arg name="set_timeout" default="false"/>
+    <arg name="timeout" default="0.0"/>
+ 
+    <!-- use Gazebo's empty_world.launch with uuv_dave_ocean_waves.world -->
+    <include file="$(find gazebo_ros)/launch/empty_world.launch">
+        <arg name="world_name" value="$(arg world_name)"/>
+        <arg name="paused" value="$(arg paused)"/>
+        <arg name="use_sim_time" value="true"/>
+        <arg name="gui" value="$(arg gui)"/>
+        <arg name="headless" value="false"/>
+        <arg name="debug" value="false"/>
+        <arg name="verbose" value="true"/>
+    </include>
+
+    <include file="$(find uuv_gazebo)/launch/rexrov_demos/rexrov_oberon_arms_demo.launch">
```

Comment:
This launch file contains a lot of the functionality.  I would propose that you refactor it so it is a bit more modular and reusable.  We can use the existing launch file as a working example and split the functionality into a set of launch files in the dave repository (or put pieces directly in this launch file)  that...

1. Configure and launch the thruster manager and controllers.
2. Configure and launch the teleoperation and launch the joint control for manipulation
  * It would be nice if there was a command line parameter to allow for swtiching between the xbox and logitech configurations.
3. Launch rviz directly from this launch file - group with the image viewer.

Reply:
As I was doing this I realized there was already a lot of modularized code available. Should be much clearer now.